instrumentation/active_record/lib/opentelemetry/instrumentation/active_record/instrumentation.rb

```diff
@@ -60,7 +60,6 @@ def patch
         end
 
         def require_dependencies
-          require 'ruby2_keywords'
```

Comment:
mmmm are we sure we can rip out this dependency i feel like there's some context here i'm missing (or have forgotten as my brain cells continue to dwindle thanks to my gas stove).

Reply:
There may be some corner cases for Ruby 2.7 so I will revert this.

Reply:
```console
🤘 bundle _2.4.3_ exec rubocop
Inspecting 24 files
......W.................

Offenses:

lib/opentelemetry/instrumentation/active_record/instrumentation.rb:63:11: W: [Correctable] Lint/RedundantRequireStatement: Remove unnecessary require statement.
          require 'ruby2_keywords'
          ^^^^^^^^^^^^^^^^^^^^^^^^
```

When I enable the rule and revert this line I get this warning.

Reply:
I need to look at the docs to see _when_ this is required. I still feel like this should be required by the _application_ using the instrumentation gems and _not_ by our gems.

Reply:
Looking at the linter rule again it is looking for a file that was _already_ required. Bundler is requiring this file for us so I am going to disable this check for that line since our AR monkey patches are using ruby2_keywords.

Reply:
Fixed a421b4500dd15ab566c464ba7a6971c2cf1b788a